uiowa/uiowa
# Allow pre tags in filtered_html text fields

Resolves #2791, relates to #3052 



# How to test


`blt ds --site=writinguniversity.uiowa.edu`
Check that config update completes.
Visit http://org-writinguniversity.local.drupal.uiowa.edu/news/2019/04/jason-r-iowa-three-boxes
See that the text is in a `<pre>` tag, allowing for poetic license in spacing (https://www.writinguniversity.org/blog/jason-r-iowa-three-boxes can be used as reference)
